gui/src/2D/advancedDynamicTexture.ts

```diff
@@ -441,40 +443,83 @@ export class AdvancedDynamicTexture extends DynamicTexture {
         }
     }
     /**
-    * Marks the texture as dirty forcing a complete update
-    */
+     * Marks the texture as dirty forcing a complete update
+     */
     public markAsDirty() {
         this._isDirty = true;
     }
     /**
-    * Helper function used to create a new style
-    * @returns a new style
-    * @see https://doc.babylonjs.com/how_to/gui#styles
-    */
+     * Helper function used to create a new style
+     * @returns a new style
+     * @see https://doc.babylonjs.com/how_to/gui#styles
+     */
     public createStyle(): Style {
         return new Style(this);
     }
     /**
-    * Adds a new control to the root container
-    * @param control defines the control to add
-    * @returns the current texture
-    */
+     * Adds a new control to the root container
+     * @param control defines the control to add
+     * @returns the current texture
+     */
     public addControl(control: Control): AdvancedDynamicTexture {
         this._rootContainer.addControl(control);
         return this;
     }
     /**
-    * Removes a control from the root container
-    * @param control defines the control to remove
-    * @returns the current texture
-    */
+     * Removes a control from the root container
+     * @param control defines the control to remove
+     * @returns the current texture
+     */
     public removeControl(control: Control): AdvancedDynamicTexture {
         this._rootContainer.removeControl(control);
         return this;
     }
     /**
-    * Release all resources
-    */
+     * Moves overlapped controls towards a position where it is not overlapping anymore.
+     * Please note that this method alters linkOffsetXInPixels and linkOffsetYInPixels.
+     * @param overlapGroup the overlap group which will be processed or undefined to process all overlap groups
+     * @param deltaStep the step size (speed) to reach the target non overlapping position (default 0.1)
+     * @param repelFactor how much is the control repelled by other controls
+     */
+    public deoverlap(overlapGroup?: number | Control[], deltaStep = 1, repelFactor = 1) {
```

Comment:
I am by no means the best at naming functions, but this function's name feels a bit off.  Might there be a better name for it?

Reply:
Yes,my terrible English lol! What about Separate, disjoint, distinct, unoverlap or something longer like makeNonOverlapping, moveToNonOverlappedPosition?